pkg/addons/data/Deployment.apps/openshift-infra/bootstrap-autoapprover.yaml

```diff
@@ -65,3 +65,5 @@ spec:
         - -c
         image: '*** GENERATED ***'
         name: signer
+      nodeSelector:
+        node-role.kubernetes.io/master: "true"
```

Comment:
@kwoodson for future PRs please can you ensure your text editor adds a trailing newline to the end of a file?

Reply:
@jim-minter, Yes I can but I'm a little confused at your earlier comment:

> please fix your editor - these trailing spaces should not be here

I removed the extra newline because of your comment.